index.html

```diff
@@ -101,15 +120,20 @@
           let y = (map_range(event.coordinate[1], extent[1], extent[3], dclExtent[1], dclExtent[3]));
           let xf = 1 * Math.floor((x));
           let yf = 1 * Math.floor((y));
-          //document.getElementById('coords').innerHTML = `(${xf}, ${yf}) / ${x}, ${y}`
-          document.getElementById('coords').innerHTML = `<span>(${xf}, ${yf})</span>`
```

Comment:
This breaks the style:
![image](https://user-images.githubusercontent.com/287189/203338434-38d4a1ca-96a5-4c35-b48e-401f05bf92e4.png)